src/db_api_util.ts

```diff
@@ -1199,6 +1199,21 @@ export async function getVoterHasVolunteer(userId: string): Promise<boolean> {
   }
 }
 
+export async function getKnownPhoneState(
+  userPhoneNumber: string
+): Promise<string | null> {
+  const client = await pool.connect();
+  try {
+    const result = await client.query(
+      'SELECT state FROM known_phone_states WHERE phone_number = $1',
+      [userPhoneNumber]
+    );
+    return result.rows.length > 0 ? result.rows[0]['state'] : null;
```

Comment:
if there aren't any rows will rows[0] be null and this throw an error?

Reply:
we check ``result.rows.length > 0 ``